src/ledger/LedgerManager.h

```diff
@@ -106,7 +106,7 @@ class LedgerManager
 
     // return the HAS that corresponds to the last closed ledger as persisted in
     // the database
-    virtual HistoryArchiveState getLastClosedLedgerHAS() = 0;
+    virtual HistoryArchiveState getPersistentHAS() = 0;
```

Comment:
why the rename?

Reply:
there are two functions with very similar names, which are easy to confuse: `HistoryManager::getLastClosedHistoryArchiveState` and `LedgerManager::getLastClosedLedgerHAS`. The former returns HAS from an in-memory LCL, the latter queries the database for whatever last HAS was saved.

Reply:
`LastClosedLedger` is used all over the code, calling it `persistent` makes it less clear on what it is

Reply:
Oh I see now what you're saying. We should remove the method in `HistoryManager` instead